tests/CMakeLists.txt

```diff
@@ -52,7 +52,6 @@ add_eosio_test_executable( unit_test
     ${CMAKE_SOURCE_DIR}/external/ethash/lib/ethash/primes.c
 )
 
-# TODO: add back eos-vm-oc once change to disable EOS VM OC subjective limits during unit test are added
-add_test(NAME consensus_tests COMMAND unit_test --report_level=detailed --color_output --run_test=evm_runtime_tests)
+add_test(NAME consensus_tests COMMAND unit_test --report_level=detailed --color_output --run_test=evm_runtime_tests -- --eos-vm-oc)
 
-add_test(NAME unit_tests COMMAND unit_test --report_level=detailed --color_output --run_test=!evm_runtime_tests)
+add_test(NAME unit_tests COMMAND unit_test --report_level=detailed --color_output --run_test=!evm_runtime_tests -- --eos-vm-oc)
```

Comment:
This means anyone performing `ctest` will now need to be using post-5.0.0 libtester otherwise the tests will fail because OC won't work properly.